android-fast-dump/src/main/cpp/CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+# For more information about using CMake with Android Studio, read the
+# documentation: https://d.android.com/studio/projects/add-native-code.html
+
+set(TARGET fast-dump)
+project(${TARGET})
+
+# Sets the minimum version of CMake required to build the native library.
+cmake_minimum_required(VERSION 3.4.1)
```

Comment:
Please add `CXX` to the `project()` invocation to avoid `C` and `ASM` compiler checks.

Reply:
done